File: testdata/jennies/rawtypes/package-with-dashes/PHPRawTypes/src/Withdashes/SomeStruct.php
<?php

namespace Grafana\Foundation\Withdashes;

class SomeStruct implements \JsonSerializable
{
    /**
     * @var mixed
     */
    public $fieldAny;

    /**
     * @param mixed|null $fieldAny
     */
    public function __construct( $fieldAny = null)
    {
        $this->fieldAny = $fieldAny ?: null;
    }

    /**
     * @param array<string, mixed> $inputData
     */
    public static function fromArray(array $inputData): self
    {
        /** @var array{FieldAny?: mixed} $inputData */
        $data = $inputData;
        return new self(
            fieldAny: $data["FieldAny"] ?? null,
        );
    }

    /**
     * @return mixed
     */
    public function jsonSerialize(): mixed
    {
        $data = new \stdClass;
        $data->FieldAny = $this->fieldAny;
        return $data;
    }
}

File: testdata/jennies/rawtypes/package-with-dashes/CSharpRawTypes/src/Grafana/Foundation/WithDashes/SomeStruct.cs
namespace Grafana.Foundation.WithDashes;


public class SomeStruct
{
    public object FieldAny;

    public SomeStruct()
    {
        this.FieldAny = new object();
    }

    public SomeStruct(object fieldAny)
    {
        this.FieldAny = fieldAny;
    }
}
